core/db/migrate/20160303150239_add_vat_country_iso_to_price.rb

```diff
@@ -0,0 +1,7 @@
+class AddVatCountryIsoToPrice < ActiveRecord::Migration
+  def change
+    add_column :spree_prices, :vat_country_iso, :string, limit: 2, null: false, default: ''
+    add_index :spree_prices, :vat_country_iso
```

Comment:
I think there's a need to have more than one price for the same things present, probably something like time-based pricing (can we make this new price the default from Wednesday next week?) 

I'd rather propose something like superseding, editable `valid_from` dates. That way, you always take the newest past valid from date, and you get a whole lot of flexibility at the cost of only one datetime field. What do you think?